### ISSUE TITLE
Consider null values the same way as undefined (revert to defaults)

### ISSUE THREAD

**Issue by webketje** [leongersen/noUiSlider#856]
**Issue reproduction** 

* Specify `options.cssClasses = undefined` vs. `options.cssClasses = null`. The former will revert to defaults and render properly, the latter will silently fail and add `undefined` classes to everything.
* Specify `options.range = { min: 0, max: undefined }` vs. `options.range = { min: 0, max: null }`
  The former throws *Missing 'min' or 'max' in 'range'*, the latter *'range' contains invalid value* 

**Proposal**
Change [options.js L433](https://github.com/leongersen/noUiSlider/blob/master/src/js/options.js#L433) `options[name] === undefined` to `options[name] === undefined || options[name] === null` (and all other options property checks where applicable*). Eventually noUiSlider could benefit from another utility like `isNumeric`:
```
function isUndef(v) {
  return v !== undefined && v !== null;
}
```
*where applicable = I might've missed a detail in the source where null is expected to behave differently from undefined, *please do point it out*. If I get enough time I'll reference this issue from a PR.

**Motivation**  
'Modern' front-end frameworks often require one to specify component properties/state up front (for reactivity). Some properties may be assigned `null`, rather than `undefined`, as per the widespread convention stated in, e.g., the accepted SO answer on [what is the difference between null and undefined in javascript](https://stackoverflow.com/questions/5076944/what-is-the-difference-between-null-and-undefined-in-javascript). When passed to noUiSlider, properties that are either `null` or `undefined` ought to be considered in the same way.

**Disclaimer**
I'm working on a (full) noUiSlider integration for Vue, and need to map parameters that are null, to undefined, for the slider to work as expected.

Thanks for reading

**Comment by leongersen** [comment]
Maybe I'm misunderstanding, but are you arguing that the plugin should throw the same exceptions for `null` and `undefined`? If so, I agree.

**Comment by webketje** [comment]
Well I'm argueing that null should be considered the same as undefined, so it's 2 things:  
1 is: throw the same exceptions (as you confirmed);  
2 is: change current behavior:  
  `options[name] === undefined ? defaults[name] : options[name]`  
  to  
  `options[name] === (undefined || null) ? defaults[name] : options[name]`

**Comment by leongersen** [comment]
Alright, cool. I'd be happy with a pull request (utility style) patching this. If you can, add a unit test or two to `tests/slider_errors.js` two show the unexpected behaviour.

The `cssPrefix` and `cssClasses` check for `undefined` themselves; they shouldn't. Instead, they should be marked required as a default is provided.

**Comment by leongersen** [comment]
I just released noUiSlider 11.1.0, which implements this. Thanks for contributing!

**Comment by webketje** [comment]
@leongersen Thanks, sorry I didn't get the chance to actually make a PR =(
